modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -1045,7 +1130,7 @@ events
 |Name |Value Type |Optional |Description
 |`adapt`
 |`CodeRef<(element: GraphElement) => NetworkAdapterType | undefined>`
-|no |
+|no |adapter to adapt element to data that can be used by the build configuration.
```

Comment:
For extension `console.topology/adapter/network`, the description for `adapt` should be 
```
adapter to adapt element to data that can be used by Networking component.
```